msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2020-05-28 12:15+0000\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: Turkish\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.1\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Language: tr\n"
"Language: tr_TR\n"

#: redbot/cogs/audio/core/events/dpy.py:34
msgid "You have attempted to run Audio's Lavalink server on an unsupported architecture. Only settings related commands will be available."
msgstr "Audio's Lavalink sunucusunu desteklenmeyen bir mimaride çalıştırmayı denediniz. Yalnızca ayarlarla ilgili komutlar kullanılabilir."

#: redbot/cogs/audio/core/events/dpy.py:68
msgid "No DJ role found. Disabling DJ mode."
msgstr "DJ rolü bulunamadı. DJ modu kapatılıyor."

#: redbot/cogs/audio/core/events/dpy.py:78
msgid "`{user_input}` is not a valid value for `{command}`"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:84
msgid "Unable To Parse Argument"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:94
#: redbot/cogs/audio/core/events/dpy.py:102
msgid "Invalid Argument"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:95
msgid "The argument you gave for `{}` is not valid: I was expecting a `{}`."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:112
msgid "Invalid Environment"
msgstr "Geçersiz Ortam"

#: redbot/cogs/audio/core/events/dpy.py:113
msgid "Connection to Lavalink has been lost."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:121
msgid "No Player Available"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:122
msgid "The bot is not connected to a voice channel."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:130
msgid "Unable to Get Track"
msgstr "Parça alınamıyor"

#: redbot/cogs/audio/core/events/dpy.py:131
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:68
msgid "Couldn't get a valid track."
msgstr "Geçerli bir parça alınamadı."

#: redbot/cogs/audio/core/events/lavalink.py:87
msgid "Auto Play started."
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:102
msgid "Now Playing"
msgstr "Şimdi oynatılıyor"

#: redbot/cogs/audio/core/events/lavalink.py:104
msgid "Track length: {length} | Requested by: {user}"
msgstr "Şarkı uzunluğu: {length} | Şarkıyı sıraya koyan: {user}"

#: redbot/cogs/audio/core/events/lavalink.py:125
msgid "Queue ended."
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:167
msgid "Multiple Errors Detected"
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:168
msgid "Closing the audio player due to multiple errors being detected. If this persists, please inform the bot owner as the Audio cog may be temporally unavailable."
msgstr "Birden çok hata nedeniyle müzik çalar kapatılıyor. Hata devam ederse, lütfen bot sahibini bilgilendiriniz. Audio eklentisi geçici olarak devre dışı kalmış olabilir."

#: redbot/cogs/audio/core/events/lavalink.py:182
msgid "Track Stuck"
msgstr "Parça Takıldı"

#: redbot/cogs/audio/core/events/lavalink.py:187
msgid "Track Error"
msgstr "Şarkı hatası"

